### PR TITLE
Completed task 4

### PR DESCRIPTION
**Pull request by 1Klevi1**
Completed task 4

- [x] ResortUI

### CODE REVIEW

File: ResortUI.java [hunk 1]
```diff
@@ -3,7 +3,7 @@
 /**
  * Write a description of class ResortUI here.
  *
- * @author (your name)
```
Comment (JackAshwell11) [inline ResortUI.java:6]: Remove the brackets.

File: ResortUI.java [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
+ * @author (Klevi)
  * @version (a version number or a date)
```
Comment (JackAshwell11) [inline ResortUI.java]: Add the current date without brackets.

File: ResortUI.java [hunk 3]
```diff
@@ -67,12 +67,14 @@ private void listAllResort() {
 
     // provide the code here  
     private void listAllPasses() {
-
```
Comment (JackAshwell11) [inline ResortUI.java:70]: Make sure this method is formatted correctly.

File: ResortUI.java [hunk 4]
```diff
@@ -67,12 +67,14 @@ private void listAllResort() {
...
+        System.out.println(fortunate.getAllPassesOnAllIslands());
     }
 
     // provide the code here 
     private void listOneIsland() {
-
+        System.out.println("Enter island name");
```
Comment (JackAshwell11) [inline ResortUI.java]: Colon after name. So like `Enter island name:"`

File: ResortUI.java [hunk 5]
```diff
@@ -67,12 +67,14 @@ private void listAllResort() {
...
+        String islandName = reader.next();
```
Comment (JackAshwell11) [inline ResortUI.java:75]: Shouldn't this be `nextLine`?

File: ResortUI.java [hunk 6]
```diff
@@ -67,12 +67,14 @@ private void listAllResort() {
...
+        System.out.println(fortunate.getAllPassesOnIsland(islandName));
```
Comment (JackAshwell11) [inline ResortUI.java:76]: Format this correctly.

File: ResortUI.java [hunk 7]
```diff
@@ -99,7 +101,12 @@ private void tryTravel() {
 
     // provide the code here
     private void travelNow() {
-
+        System.out.println("Enter pass id");
```
Comment (JackAshwell11) [inline ResortUI.java]: Add colon after id like above.

File: ResortUI.java [hunk 8]
```diff
@@ -99,7 +101,12 @@ private void tryTravel() {
...
+        int trav = reader.nextInt();
+        reader.nextLine();
+        System.out.println("Enter ferry code");
```
Comment (JackAshwell11) [inline ResortUI.java]: Enter colon after.

File: ResortUI.java [hunk 9]
```diff
@@ -111,7 +118,11 @@ private void viewPass() {
 
     // provide the code here
     private void updateCredits() {
-
+        System.out.println("Please enter the id of the pass toping up their credits");
```
Comment (JackAshwell11) [inline ResortUI.java]: Colon after.

File: ResortUI.java [hunk 10]
```diff
@@ -111,7 +118,11 @@ private void viewPass() {
...
+        int passId= reader.nextInt();
```
Comment (JackAshwell11) [inline ResortUI.java]: Space after passId.

File: ResortUI.java [hunk 11]
```diff
@@ -111,7 +118,11 @@ private void viewPass() {
...
+        System.out.println("Please enter the number of credits to be added");
```
Comment (JackAshwell11) [inline ResortUI.java]: Colon after.

**Review by JackAshwell11 — CHANGES_REQUESTED** [review]

File: ResortUI.java (hunk 1 above)
Comment (1Klevi1) [reply ResortUI.java:6]: done

File: ResortUI.java (hunk 2 above)
Comment (1Klevi1) [reply ResortUI.java]: done

File: ResortUI.java (hunk 3 above)
Comment (1Klevi1) [reply ResortUI.java:70]: okk

File: ResortUI.java (hunk 4 above)
Comment (1Klevi1) [reply ResortUI.java]: done

File: ResortUI.java (hunk 5 above)
Comment (1Klevi1) [reply ResortUI.java:75]: done

File: ResortUI.java (hunk 7 above)
Comment (1Klevi1) [reply ResortUI.java]: done

File: ResortUI.java (hunk 9 above)
Comment (1Klevi1) [reply ResortUI.java]: done

File: ResortUI.java (hunk 10 above)
Comment (1Klevi1) [reply ResortUI.java]: done

File: ResortUI.java (hunk 11 above)
Comment (1Klevi1) [reply ResortUI.java]: done

**Review by JackAshwell11 — APPROVED** [review]

**Pull request merged**